src/site/content/en/blog/badging-api/index.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Badging for app icons
+subhead: The Badging API allows installed web apps to set an application-wide badge, shown in an operating-system-specific place associated with the application, such as the shelf or home screen.
+authors:
+  - petelepage
+description: The Badging API allows installed web apps to set an application-wide badge, shown in an operating-system-specific place associated with the application, such as the shelf or home screen. Badging makes it easy to subtly notify the user that there is some new activity that might require their attention, or it can be used to indicate a small amount of information, such as an unread count.
+date: 2018-12-11
+updated: 2019-10-08
+tags:
+  - post
+  - capabilities
+  - badging
+  - progressive-web-apps
+  - notifications
+  - origintrials
+  - fugu
+hero: hero.jpg
+alt: Phone showing several notification badges
+---
+
+{% Aside 'caution' %}
+  This API is part of the new
+  [capabilities project](https://developers.google.com/web/updates/capabilities),
+  and starting in Chrome 73 it is available as an [**origin trial**](#ot).
+  This post will be updated as the Badging API evolves.
+{% endAside %}
+
+## What is the Badging API? {: #what }
+
+<figure class="w-figure w-figure--inline-right" style="max-width:50%">
```

Comment:
In general we try to discourage inline styles because when we update the components it may conflict with the inline styles folks have created.

In this case I think the image itself should be resized to around 400px. I think it will then display at the correct size on desktop and mobile.

Reply:
I used this because I couldn't find an equivalent to `.attempt-right` with from DevSite. Is there one that I missed? If not, can we get one?

Reply:
N/M, removed the `max-width` attribute